web/src/main/java/co/mobileaction/example/web/controller/PostController.java

```diff
@@ -34,11 +35,18 @@ public ResponseEntity<List<Post>> getPosts(@PageableDefault(size = 10)
         return ResponseEntity.ok(postService.findPosts(pageable));
     }
 
-    @DeleteMapping("{postId}")
+    @DeleteMapping("/{postId}")
     public ResponseEntity<Boolean> deletePost(@PathVariable Long postId)
     {
+        //postService.deletePost(postId);
         postService.deletePost(postId);
+        return ResponseEntity.ok(true);
+    }
 
+    @DeleteMapping("/user/{userId}")
+    public ResponseEntity<Boolean> deletePostsByUser(@PathVariable Long userId)
+    {
+        postService.deletePost(userId);

```

Comment:
this will not delete posts of user, this method seems to take `postId`. You need to create a new method to take `userId` rather than `postId`, and it should delete all the posts of the requested `userId`. Remember that, there can be more than one post of the requested user.